app.js

```diff
@@ -0,0 +1,104 @@
+var Issue = can.DefineMap.extend({
+  seal: false
+}, {
+  id: 'number',
+  title: 'string',
+  sort_position: 'number',
+  body: 'string'
+});
+
+Issue.List = can.DefineList.extend({
+  '*': Issue
+});
+
+Issue.algebra = new can.set.Algebra(
```

Comment:
It doesn't really make a difference, but you don't need to set the algebra on the `Issue` Type. Adding it to the connection (via `superMap({ })`) is enough.